File: demo/src/Demos/Fable_demo.js
import { createAtom } from "fable-core/Util";
export const counterBtn = document.querySelector("#counter-btn");
export const counterResult = document.querySelector("#counter-result");
export let count = createAtom(0);

counterBtn.onclick = function (ev) {
  count(count() + 1);
  counterResult.innerHTML = "You clicked " + count().toString() + " times";
};

File: demo/src/Demos/Fable_demo.fs
module App.Demo

open Fable.Import.Browser

let counterBtn = document.querySelector("#counter-btn") :?> HTMLButtonElement
let counterResult = document.querySelector("#counter-result")
let mutable count = 0

counterBtn.onclick <-
    fun _ ->
        count <- count + 1
        counterResult.innerHTML <- "You clicked " + string count + " times"

open Fable.Core.JsInterop

let add (x:int) (y:int) : int = import "add" "./Demos/my-lib.js"

add 2 3
|> console.log